homeassistant/components/bond/utils.py

```diff
@@ -224,4 +226,15 @@ def devices(self) -> list[BondDevice]:
     @property
     def is_bridge(self) -> bool:
         """Return if the Bond is a Bond Bridge."""
+        target = self.target
+        if target and any(
```

Comment:
I wish there was a more deterministic way to figure this out but I can't come up with one

Reply:
The official way that we have to figure that out is by looking at the serial/bond ID.

```kotlin
// Bond Bridges
BRIDGE_SNOWBIRD -> "^[A-C]\\w*\$"
BRIDGE_ZERMATT -> "^Z(Z|X)\\w*\$"
BRIDGE_PRO -> "^ZP\\w*\$"
// SBBs
SBB_CEILING_FAN -> "^K\\w*\$"
SBB_LIGHTS -> "^T\\w*\$"
SBB_PLUG -> "^P\\w*\$"
```
But I think maybe is best to have `isBridge()` and `isSBB()` functions on bond-cli... I'll work on that.

Note that is a few SBB Lights that have more than one device and need the bridge endpoint to adjust the `bluelight` property and such



Another possible way to look at this is to check if the target starts with `snowbird` or `zermatt`, which would include the `zermatt-pro` target. Anything else could be considered a SBB (at least for now)

Reply:
Looks like if it starts with K T or P it’s SBB

I was really hoping to avoid the magic strings. I’ll push another turn of this later tonight with that logic instead if I don’t run out of time.

Reply:
Thanks for the info @marciogranzotto

Reply:
Also I don’t have the lights or plugs for testing. Any idea where I can pick those up in the us market?

Reply:
Also if you are adding the isSBB to the lib we should use that instead. I’m not sure if the bond-cli uses the same lib or not and I’m mobile so it difficult to check

Reply:
Also since you are in a position to turn the lib and if that's the intent, I'm happy to close this PR if you want to make the change.

Reply:
ah sorry, I meant `bond-async`!
Ok, you can close it. I'm working on the change on bond-async now